src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -3216,4 +3216,13 @@ You can manage this in Preferences -&gt; Security.</value>
   <data name="TrustLocationSkippedNotification" xml:space="preserve">
     <value>No trusted location has been added.</value>
   </data>
+  <data name="MessagePackageTargetOtherHosts2" xml:space="preserve">
+    <value>The package or one of its dependencies targets a different environment, such as Revit, Civil 3D, Advance Steel, Alias or FormIt. This can cause instability and unexpected problems.</value>
+  </data>
+  <data name="MessagePackageTargetOtherHostShort" xml:space="preserve">
+    <value>Package targets a different host than current host application.</value>
+  </data>
+  <data name="TitlePackageTargetOtherHost" xml:space="preserve">
+    <value>Package Host Error</value>
+  </data>
```

Comment:
Aren't one or more of these messages already added previously here: https://github.com/DynamoDS/Dynamo/pull/12927?

Reply:
no, they are slightly different, @sm6srw's messages include a question - do you want to proceed? etc - since this is a notification and not a message box, and we've already loaded the package, that did not make sense to me.